mlflow/store/file_store.py

```diff
@@ -172,8 +177,9 @@ def list_experiments(self, view_type=ViewType.ACTIVE_ONLY):
 
     def _create_experiment_with_id(self, name, experiment_id, artifact_uri):
         self._check_root_dir()
-        meta_dir = mkdir(self.root_directory, experiment_id)
-        artifact_uri = artifact_uri or build_path(self.artifact_root_uri, experiment_id)
+        meta_dir = mkdir(self.root_directory, str(experiment_id))
+        artifact_uri = artifact_uri or path_to_local_file_uri(
```

Comment:
is this where you said if a user supplies the artifact_uri that is local we don't covert to `file:/...`? i guess the only thing i see that will be weird is `get_artifact_uri` will return a weirdly-formatted uri like `c:\user\given\artifact\uri/relative/path` which should be ok...

Reply:
Yes, we do not convert user supplied artifact uri. 

Windows absolute path would not work though as it would look like a uri with single character scheme. For absolute paths on windows, user must use the file:///<drive>:/<path> pattern. 

In case of relative path with '\' separator, we do not convert it and mixed separators should work in windows.